pkg/core/transaction.go

```diff
@@ -34,13 +34,54 @@ func (t *TransactionData) Reverse() TransactionData {
 
 var _ json.Marshaler = Transaction{}
 
+type RawTransaction struct {
```

Comment:
I also feel like we should avoid creating this new struct. Why do you need it?

Reply:
The "RawTransaction" is the struct used to store transactions at logs level.
Basically, this struct contains fields required to rebuild the model reading only the logs.
So, it excludes pre/post commit volumes, and potentially, futures fields we will add.

Reply:
We decided together to try embedding a new struct without volumes in the `Transaction` struct.